Basic/ClientDriven/Assets/Scenes/Bootstrap.unity

```diff
@@ -123,83 +123,99 @@ NavMeshSettings:
     debug:
```

Comment:
Let's move the spawner objects to the same spot in the hierarchy as before, above the `----Art----` separator GameObject (and also you could re-tag this GameObject as "EditorOnly" as it's not currently).

Reply:
Hey, I just saw this. Apologies if it wasn't too clear, but I had meant to apply that to the `-------- Art --------` GameObject.
If this is applied to the spawners, the GameObjects be excluded from builds.
See more about tags here: https://docs.unity3d.com/Manual/Tags.html.

Reply:
Oooh, ok that makes sense, thank you for the clarification. I will add it to the -------- Art -------- GameObject.

Reply:
Last item (I swear) is that the Ingredient spawners in the scene still have NetworkObject components on them, that are not needed.

You can inspect each one and through the inspector just revert each addition:
<img width="364" alt="Screenshot 2024-04-15 at 16 25 27" src="https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/assets/75813458/46a17d3c-51b9-4ab7-aeba-d8ea07d88767">
One is for the tag and the other for the added NetworkObject component.

Reply:
^ The tag can still be reverted!